src/Conference/AddConferenceForm.jsx

```diff
@@ -24,7 +25,7 @@ const Form = () => {
       dateTime: conference.dateTime + ":00Z"
     };
     try {
-      await createNewConference(newConference);
+      await createNewConference(newConference, allCookies.sessionToken.token);
```

Comment:
I would remove cookies from the file completely and move into API (you might have to stop using hooks for that). This would save on complexity of components and centralise knowledge of the cookies into one place again, which you have lost in this refactor.

Reply:
I've taken a note of this, we'll bring it up in refinement and probably add it to a new ticket as it it out of scope for this branch